boltzrpc/boltzrpc.proto

```diff
@@ -497,6 +530,58 @@ message CreateReverseSwapResponse {
     optional string invoice = 5;
 }
 
+message CreateChainSwapRequest {
+    // amount of satoshis to swap
+    uint64 amount = 1;
```

Comment:
Is this the equivalent to `userLockAmount`? Not clear from the comment

Reply:
Yes. Should we expose both user and server lock amount in the api?

Reply:
Our just write in the comment that it is the amount you are expected to send